src/main/java/me/dinowernli/grpc/polyglot/command/ServiceList.java

```diff
@@ -113,4 +124,33 @@ private static String renderDescriptor(FieldDescriptor descriptor, String indent
       return fieldPrefix + ": " + descriptor.getJavaType();
     }
   }
-}
+
+  private static void printJsonOutput(Output output, ServiceResolver serviceResolver, Optional<String> serviceFilter) {
+    output.writeLine("{ \"serviceProtos\": [");
```

Comment:
We should probably not be constructing json ourselves here. If we want to wrap the existing protobuf output, we should define a proto which wraps the messages we need and print that a single time (this would also obviate the need for changing the messageprinter).